### DISCUSSION TITLE
How to remove buttons / tokens

### DISCUSSION THREAD

**Discussion by Faelyke** [TerriaJS/terriajs#5269] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74516286/108953531-7513e000-76bf-11eb-96e2-175f2ea2618c.png)

I wish to remove these icons from the map. However the config/json doesn't appear to list them. Additionally:

"parameters": {
        "autoPlay": false,
}

Setting autoPlay to false doesn't appear to be working when set in the parameters. Could use some assistance.

**Comment by tephenavies** [comment, marked as answer]
Hi @Faelyke. Currently these icons can be removed by modifying `lib/Views/UserInterface.jsx` in TerriaMap. If you need to remove a lot of them you can copy `lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx` from `terriajs` into TerriaMap's `UserInterface.jsx` and make major changes (it's a bit tedious because you need to change all the imports to "terriajs/lib/...", but it should work).

There is a PR that should be merged soon https://github.com/TerriaJS/terriajs/pull/5131 that will allow better control in config files over the buttons.

**Reply by Faelyke** [reply]
Thanks for the help. I'll give it a try tomorrow.
